src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanEnterpriseBuildStep.java

```diff
@@ -788,6 +974,50 @@ public FormValidation doCheckJobName(@QueryParameter String jobName) {
 			return FormValidation.validateRequired(jobName);
 		}
 
+		public FormValidation doCheckExploreDataTestScan(@QueryParameter String exploreDataTestScan) {
+			return FormValidation.validateRequired(exploreDataTestScan);
+		}
+
+		public FormValidation doCheckPostmanCollectionFile(@QueryParameter String postmanCollectionFile) {
+			if(postmanCollectionFile != null && !postmanCollectionFile.isEmpty()) {
```

Comment:
Rather than a nested "if" block, this should just be logically "anded" with the other conditions.

Reply:
Okay, sure.